### DISCUSSION TITLE
mspider/user-guide/traffic-governance/gateway-rules/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1272] [Announcements]
# mspider/user-guide/traffic-governance/gateway-rules/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/user-guide/traffic-governance/gateway-rules/

**Comment by liyuerich** [comment]
能否展示一下怎么添加服务端

**Comment by liyuerich** [comment]
新版本还有规则标签，需要展示

**Comment by samzong** [comment]
网关规则创建页面，内容与配图需要更新。

#2784
